src/widgets/splits/Split.cpp

```diff
@@ -765,6 +766,47 @@ void Split::addShortcuts()
              }
              return "";
          }},
+        {"popupOverlay",
+         [this](const auto &) -> QString {
+             this->showOverlayWindow();
+             return {};
+         }},
+        {"togglePopupInertia",
+         [this](const auto &args) -> QString {
+             if (args.empty())
+             {
+                 return "No arguments provided to togglePopupInertia (expected "
+                        "one)";
+             }
+             const auto &arg = args.front();
+
+             if (arg == "this")
+             {
+                 if (this->overlayWindow_)
+                 {
+                     this->overlayWindow_->toggleInertia();
+                 }
+                 return {};
+             }
+             if (arg == "thisOrAll")
+             {
+                 if (this->overlayWindow_)
+                 {
+                     this->overlayWindow_->toggleInertia();
+                 }
+                 else
+                 {
+                     getApp()->getWindows()->toggleAllPopupInertia();
+                 }
+                 return {};
+             }
+             if (arg == "all")
+             {
+                 getApp()->getWindows()->toggleAllPopupInertia();
+                 return {};
+             }
```

Comment:
I think the default keybind action should be `all`, and I think `toggleAllPopupIntertia` should not toggle each overlay individually, rather think of all of them as a group.
If any overlay is interactable, make all overlays inert. Otherwise, make all overlays interactable.